dkg/dkg.go

```diff
@@ -674,3 +680,15 @@ func writeLockToAPI(ctx context.Context, publishAddr string, lock cluster.Lock)
 
 	return nil
 }
+
+// validateKeymanagerFlags returns an error if one keymanager flag is present but the other is not.
+func validateKeymanagerFlags(addr, authToken string) error {
+	if addr != "" && authToken == "" {
+		return errors.New("keymanager address provided but authentication token absent")
```

Comment:
make explicit that this is configuratio flag issue. Suggest `--keymanager-address provided but --keymanager-auth-token absent, Please fix configuration flags`

Reply:
okay, also added a test for this function